src/panel/pages/timeline/components/TimelineRow.tsx

```diff
@@ -16,6 +20,7 @@ export const TimelineRow: FC<{ events: ParsedEvent[] }> = ({ events }) => {
           <TimelineEvent
             key={e.key}
             event={e}
+            selectEvent={() => setSelectedEvent(e)}
```

Comment:
We can skip forwarding this prop and just access it via context in the TimelineEvent component

Reply:
I was thinking that it might be nice to keep TimelineEvent without knowledge of the context?

Because of the closure I was hoping it might not _quite_ be considered prop forwarding

Reply:
Sure, it's only passed down to one component anyhow 👍